src/main/java/com/noroff/mefit/data/model/Program.java

```diff
@@ -0,0 +1,27 @@
+package com.noroff.mefit.data.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Table(name = "program", schema = "public")
+public class Program {
+    @Id
+    @Column
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
```

Comment:
`GET` operations or finding by ID results in an 500 error.
```json
{
  "timestamp": "2022-03-14T13:12:23.649+00:00",
  "status": 500,
  "error": "Internal Server Error",
  "path": "/api/program/2"
}
```
This change fixes the error:
```suggestion
    private Long id;
```